examples/Huggingface_Transformers/Transformer_handler_generalized.py

```diff
@@ -75,12 +78,19 @@ def initialize(self, ctx):
                 self.model = AutoModelForCausalLM.from_pretrained(model_dir)
             else:
                 logger.warning("Missing the operation mode.")
-            self.model.to(self.device)
+
+            if self.setup_config["model_parallel"] and "gpt2" in self.setup_config["model_name"]:
```

Comment:
why `"gt2" in ...` are there multiple gpt like models supported? if so would be useful to list them out in the readme

Reply:
Done